### PR TITLE
Add support for blockHandlers and callHandlers

### LINKED ISSUE #223: Update subgraph validation to block and transaction triggers
* Validate these fields
* Decide on the data to pass down to handlers

### PR DESCRIPTION
**Pull request by Jannis**
Resolves #223.

This adds `blockHandlers` and `callHandlers` the subgraph manifest schema and includes them in the subgraph validation.

Open question: Should we introduce a new mapping kind alias called `ethereum/contract` for `ethereum/event`, since we're now supporting more than events?

### CODE REVIEW

**Comment by Jannis** [comment]
Tests are only failing because of Node 11 in Travis acting up. The Node 10 tests pass fine.

File: manifest-schema.graphql [hunk 1]
```diff
@@ -27,14 +27,25 @@ type EthereumContractMapping {
...
 type EthereumContractAbi {
   name: String!
   file: File!
 }
 
+type EthereumBlockHandler {
+  handler: String!
+}
+
+type EthereumTransactionHandlers {
```
Comment (olibearo) [inline manifest-schema.graphql]: You mentioned in Slack this should be singular, not plural. Should we also make the change for `EthereumContractEventHandlers`?

File: manifest-schema.graphql [hunk 2]
```diff
@@ -27,14 +27,25 @@ type EthereumContractMapping {
   file: File!
   entities: [String!]!
   abis: [EthereumContractAbi!]!
-  eventHandlers: [EthereumContractEventHandlers]!
+  blockHandlers: EthereumBlockHandler
```
Comment (olibearo) [inline manifest-schema.graphql]: Also, maybe this should be singular as well since it is not an array?

Comment (Jannis) [reply manifest-schema.graphql]: Yep, good point

File: manifest-schema.graphql (hunk 1 above)
Comment (Jannis) [reply manifest-schema.graphql]: Yeah, that makes sense!

**Comment by Jannis** [comment]
@olibearo This is ready. Do you want to take another look?

**Review by ghost — APPROVED** [review]

**Pull request merged**